opencga-app/app/scripts/docker/build-all.sh

```diff
@@ -113,9 +113,9 @@ if [ ! -d "${buildPath}" ]; then
     docker run -it --rm \
     -v "$PWD":/src \
     -v "$HOME/.m2":/root/.m2 \
-    -w /src maven:3.6-jdk-8 \
-    mvn -T 1C install \
-    -DskipTests -Dstorage-hadoop -Popencga-storage-hadoop-deps -P"${HADOOP_FLAVOR}" -DOPENCGA.STORAGE.DEFAULT_ENGINE=hadoop -Dopencga.war.name=opencga
+    -w /src maven:3.5.3-jdk-8 \
+    mvn clean install -U \
```

Comment:
@j-coll I added -U only due the issue I ran into with biodata not compiling - is it safe/wise to leave this in?

Reply:
@j-coll